### ISSUE TITLE
Empty .gitignore on symfony new project

### ISSUE THREAD

**Issue by mariuspuiu** [symfony/symfony-installer#242]
New projects created via the installer have empty `.gitignore` file in the root of the app by default.

![image](https://cloud.githubusercontent.com/assets/878197/13881511/db6509a8-ed29-11e5-9509-488d175d2c09.png)

Meaning that someone might add by mistake the `cache`, `logs` and others to the repository:

![image](https://cloud.githubusercontent.com/assets/878197/13879578/dc943924-ed20-11e5-981c-c0e4f18f36c5.png)

**Comment by Pierstoval** [comment]
According to your `symfony new my_project` you should have run `ls my_project/.gitignore`, and it would have worked actually :confused: 

I just tested on my machine and it works.

**Comment by mariuspuiu** [comment]
Sorry, forgot to cd to my_project. Still, the .gitignore file is empty.

**Comment by mariuspuiu** [comment]
It seems that I have an issue on my system, I removed the `@` in front of `file_get_contents()` (`DownloadCommand.php`, line `326`) and the output is:
```
PHP Warning:  file_get_contents(https://raw.githubusercontent.com/symfony/symfony-standard/v3.0.3/.gitignore): failed to open stream: Connection timed out in /var/www/symfony-installer/src/Symfony/Installer/DownloadCommand.php on line 326
```

With cURL, it works:
```
curl https://raw.githubusercontent.com/symfony/symfony-standard/v3.0.3/.gitignore
/app/config/parameters.yml
/build/
/phpunit.xml
/var/*
!/var/cache
/var/cache/*
!var/cache/.gitkeep
!/var/logs
/var/logs/*
!var/logs/.gitkeep
!/var/sessions
/var/sessions/*
!var/sessions/.gitkeep
!var/SymfonyRequirements.php
/vendor/
/web/bundles/
```

**Comment by Pierstoval** [comment]
Check your PHP configuration:
* Do you have `openssl` module installed and enabled?
* Is `http` registered as a stream?
* Which version do you use?

**Comment by mariuspuiu** [comment]
```
Registered Stream Socket Transports	tcp, udp, unix, udg, ssl, sslv3, sslv2, tls, tlsv1.0, tlsv1.1, tlsv1.2
```
```
OpenSSL support enabled
OpenSSL Library Version OpenSSL 1.0.1e-fips 11 Feb 2013
```

**Comment by brightdroid** [comment]
@mariuspuiu are you also behind a proxy? Then it's definitely the same problem as #243.

**Comment by mariuspuiu** [comment]
@brightdroid yes, I am behind a proxy, the `http_proxy` and `https_proxy` environment variables are set ok.

**Comment by brightdroid** [comment]
Exactly this is the problem, the installer does not use the env. vars, cause it uses PHP's "file_get_contents" function in some places, which can't handle proxies.

**Comment by stof** [comment]
Given that we already have a dependency on guzzle, we should use it everywhere instead of using ``file_get_contents`` directly in some cases. It would solve these issues. Any volunteer to provide a pull request ?

**Comment by mariuspuiu** [comment]
For this specific issue, I did a pull request.

**Comment by javiereguiluz** [comment]
Closing it as fixed by #245. @mariuspuiu thanks for solving this. It will be available in the next stable release of the installer.
